### ISSUE TITLE
coverity-cov-cpp.xml

### ISSUE THREAD

**Issue by HassanQayyum1993** [coverity/coverity-sonar-plugin#30]
HI,
Are rules description mentioned in file coverity-cov-cpp.xml, given by Coverity?

**Comment by jbriggs22** [comment]
@HassanQayyum1993 , yes the xml rules documentation was generated using a Coverity utility.
